### PR TITLE
Improve Azure documentation

### PR DESCRIPTION
**Pull request by otetard**

### CODE REVIEW

File: docs/integrations/azure_ad.md [hunk 1]
```diff
@@ -2,47 +2,86 @@ uuid: 19cd2ed6-f90c-47f7-a46b-974354a107bb
...
 PS Azure:\> az eventhubs namespace create --name company-eventhub --resource-group company-resource-group --enable-kafka true --subscription uuid
+```
 
+```powershell
 PS Azure:\> az eventhubs eventhub create --resource-group company-resource-group --namespace-name company-eventhub --name active-directory-event --message-retention 3 --partition-count 4 --subscription uuid
 ```
 
-Navigate to: `Home > Event Hubs > company-eventhub - Shared access policies`. From there, you can create a policy (e.g. RootManageSharedAccessKey) with the claims `Manage`, `Send` and `Listen`, and note the `Primary Key` that will be used as the `SharedAccessKey`.
+!!! info
+    Please replace the `uuid` mention with your subscription ID retrieved previously (see below).
```
Comment (goudyj) [inline docs/integrations/azure_ad.md:35]:
```suggestion
    Please replace :
    
    * `company-resource-group` with the name of your resourceGroup
    * `uuid` with your subscription ID retrieved previously (see below).
```

File: docs/integrations/azure_ad.md [hunk 2]
```diff
@@ -2,47 +2,86 @@ uuid: 19cd2ed6-f90c-47f7-a46b-974354a107bb
...
+```powershell
+PS Azure:\> az storage account create --name "company-sekoiaiocheckpoint" --resource-group "company-resource-group"
+```
+
+```powershell
+PS Azure:\> az storage container create --name "active-directory-event" --account-name "company-sekoiaiocheckpoint"
+```
+
+!!! info
+    The container name, here `active-directory-event` should be the same as the Event Hub’s one.
```
Comment (goudyj) [inline docs/integrations/azure_ad.md:58]:
```suggestion
    The container name, here `active-directory-event` should be the same as the Event Hub’s one.
    You also need to replace `company-resource-group` with the name of your resourceGroup
```

File: docs/integrations/azure_ad.md [hunk 3]
```diff
@@ -2,47 +2,86 @@ uuid: 19cd2ed6-f90c-47f7-a46b-974354a107bb
...
 
-Theses changes have to be made from the Azure web portal (https://portal.azure.com).
+Theses changes have to be made from the [Azure Web Portal](https://portal.azure.com).
+
+### Azure Event Hubs
 
-### Event hubs
-As a prerequisite you need an `Event Hubs` (e.g. company-eventhub) and to choose an existing `resourceGroup` or create a new one (e.g. company-resource-group).
-You also need your `Subscription ID` if you don't have a default one.
+As a prerequisite, you need an “Event Hubs” (e.g. `company-eventhub`) and to choose an existing `resourceGroup`, or create a new one (e.g. `company-resource-group`).
```
[the remark this answers is not in the document]
Comment (otetard) [reply docs/integrations/azure_ad.md]: Change applied, thanks!

File: docs/integrations/azure_ad.md [hunk 4]
```diff
@@ -2,47 +2,86 @@ uuid: 19cd2ed6-f90c-47f7-a46b-974354a107bb
...
 
-Navigate to: `Home > Cost Management + Billing > Subscriptions`. From there, copy the relevant `Subscription ID` that will be used in the command line (e.g. uuid)
+#### Retrieve your Subscription ID
 
-Then you use Azure powershell (within Cloud Shell interface for example): you will a create a global `Event Hubs`, then specific `Event Hub` (e.g. active-directory-event).
+You also need your “Subscription ID” if you don't have a default one. In Azure Web Portal, navigate to: “Home”, “Cost Management + Billing”, ”Subscriptions”. From there, copy the relevant “Subscription ID” that will be used in the command line (e.g. `uuid`)
+
+#### Create the Event Hubs
+
+Then, use Azure PowerShell (within Cloud Shell interface for example), to a create a global “Event Hubs”, then a specific `Event Hub` (e.g. `active-directory-event`):
```
[the remark this answers is not in the document]
Comment (otetard) [reply docs/integrations/azure_ad.md]: Change applied, thanks!

File: docs/integrations/azure_ad.md (hunk 1 above)
Comment (otetard) [reply docs/integrations/azure_ad.md]: Change applied, thanks!

File: docs/integrations/azure_ad.md (hunk 2 above)
Comment (otetard) [reply docs/integrations/azure_ad.md:60]: Change applied, thanks!

**Review by goudyj — APPROVED** [review]

**Pull request merged**